install_config/aggregate_logging.adoc

```diff
@@ -3,10 +3,9 @@
 {product-author}
 {product-version}
 ifdef::openshift-enterprise[]
-:latest-tag: v3.11.170
-endif::[]
```

Comment:
Was removing this first `endif::[]` intentional?

Reply:
whoa! not sure how that happened! thanks, fixing now.